src/segment-loader.js

```diff
@@ -2098,7 +2098,7 @@ export default class SegmentLoader extends videojs.EventTarget {
       this.playlistOfLastInitSegment_[type] = playlist;
       // we should only be appending the next init segment if we detect a change, or if
       // the segment has a map
-      this.appendInitSegment_[type] = map ? true : false;
+      this.appendInitSegment_[type] = false;
```

Comment:
Maybe we should update the comment to something like: "disable future init segments appends for this type (until a change is necessary)"